.github/workflows/release.yaml

```diff
@@ -86,3 +86,43 @@ jobs:
           asset_path: ${{ env.ASSET }}
           asset_name: ${{ env.ASSET }}
           asset_content_type: application/gzip
+  publish-docker:
+    name: publish-docker
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Authenticate with github container registry
+        run: echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
+
+      - name: Build elixir image
+        run: docker build --file Dockerfile.elixir --tag gleam-elixir .
+
+      - name: Build erlang image
+        run: docker build --file Dockerfile.erlang --tag gleam-erlang .
+
+      - name: Push images
+        run: |
+          IMAGE_ID_BASE=ghcr.io/${{ github.repository }}/gleam
+
+          # Change all uppercase to lowercase
+          IMAGE_ID_BASE=$(echo $IMAGE_ID_BASE | tr '[A-Z]' '[a-z]')
+
+          # Strip git ref prefix from version
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+
+          # Strip "v" prefix from tag name
+          [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
```

Comment:
A bunch of this code just turns a tag of `v0.x.y` into a docker tag of `0.x.y`, which is the convention for docker images.